### ISSUE TITLE
Error when trying to open new form from Search Bar

### ISSUE THREAD

**Issue by umairsy** [frappe/erpnext#7797]
![new-search](https://cloud.githubusercontent.com/assets/910238/23153662/c31d87dc-f82f-11e6-8819-046f0c6986db.gif)

```
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 56, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 19, in handle
    execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 42, in execute_cmd
    ret = frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 903, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 70, in getdoctype
    docs = get_meta_bundle(doctype)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/load.py", line 81, in get_meta_bundle
    bundle = [frappe.desk.form.meta.get_meta(doctype)]
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 20, in get_meta
    meta = frappe.cache().hget("form_meta", doctype, lambda: FormMeta(doctype))
  File "/home/frappe/frappe-bench/apps/frappe/frappe/utils/redis_wrapper.py", line 171, in hget
    value = generator()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 20, in 
    meta = frappe.cache().hget("form_meta", doctype, lambda: FormMeta(doctype))
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 32, in __init__
    self.load_assets()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 42, in load_assets
    self.load_templates()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/form/meta.py", line 151, in load_templates
    module = load_doctype_module(self.name)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/modules/utils.py", line 179, in load_doctype_module
    raise ImportError, 'Module import failed for {0} ({1})'.format(doctype, module_name)
ImportError: Module import failed for Sales Order (erpnext.selling.doctype.sales_order.sales_order)
```

**Comment by umairsy** [comment]
Same error is occurring when trying to access List View or Report from the awesome bar.

**Comment by pratu16x7** [comment]
fixed via frappe/frappe#2768
